draft-opsarea-rfc5706bis.md

```diff
@@ -1025,15 +1025,18 @@ Information Models and Data Models
    discontinuities in counter?  Does this make any difference in
    evaluating the performance of a protocol?
 
-   The protocol document should make clear the limitations implicit
-   within the protocol and the behavior when limits are exceeded.  This
-   should be considered in a data-modeling-independent manner -- what
-   makes managed-protocol sense, not what makes management-protocol-
-   sense.  If constraints are not managed-protocol-dependent, then it
-   should be left for the management-protocol data modelers to decide.
-   For example, VLAN identifiers have a range of 1..4095 because of the
-   VLAN standards.  A MIB implementing a VLAN table should be able to
-   support 4096 entries because the content being modeled requires it.
+   The protocol specification should clearly define any inherent
+   limitations and describe expected behavior when those limits
+   are exceeded. These considerations should be made independently
+   of any specific management protocol or data modeling language.
+   In other words, focus on what makes sense for the protocol being
+   managed, not the protocol used for management.  If a constraint
```

Comment:
"protocol used for management" could be confusing at first. Maybe specify (e.g., SNMP, NETCONF/YANG)

Reply:
I do not see how "protocol used for management" can be confusing, particularly c.f. "protocol being managed".